src/components/UserInfo.js

```diff
@@ -1,18 +1,24 @@
 export default class UserInfo {
-  constructor({ userName, userDescription }) {
+  constructor({ userName, userDescription, currentProfPic }) {
     this._name = document.querySelector(userName);
-    this._description = document.querySelector(userDescription);
+    this._about = document.querySelector(userDescription);
+    this._avatar = document.querySelector(currentProfPic);
   }
 
   getUserInfo() {
     return {
       name: this._name.textContent,
-      description: this._description.textContent,
+      description: this._about.textContent,
+      avatar: this._avatar,
     };
   }
 
   setUserInfo(name, description) {
     this._name.textContent = name;
-    this._description.textContent = description;
+    this._about.textContent = description;
+  }
+
+  setUserAvatar(avatar) {
```

Comment:
It's great to have a setter for the avatar, since it's controlled by a different form